conda_build/cli/main_index.py

```diff
@@ -60,11 +60,23 @@ def parse_args(args):
         "--no-progress", help="Hide progress bars", action="store_false", dest="progress"
     )
     p.add_argument(
-        "--convert",
```

Comment:
Is the removal of the `--convert` option intentional?

Reply:
Yeah, I think that's better done as an explicit step with CPH.  No reason to include it here.